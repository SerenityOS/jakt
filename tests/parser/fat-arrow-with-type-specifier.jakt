class S {
    public function get_answer() -> i32 => 42
}

function greeting() -> String => "Hello"

function main() {
    println("{} friends! The answer is {}", greeting(), S::get_answer())
}
