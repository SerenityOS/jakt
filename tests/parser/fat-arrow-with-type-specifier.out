Hello friends! The answer is 42
